### PR TITLE
Updated Filter section

### PR DESCRIPTION
**Pull request by kentico-anthonym**
Added an additional example and split the filters between those that accept only two parameters and those that accept more than two.

### CODE REVIEW

File: doc/delivery.md [hunk 1]
```diff
@@ -195,6 +195,14 @@ deliveryClient.items<Movie>()
   .getObservable()
   .subscribe(response => console.log(response));
 ```
+The following filters accept multiple query arguments within a list: `inFilter`, `allFilter`, `anyFilter`, `containsFilter`.
+```typescript
+deliveryClient.items<Movie>()
+  .type('movie')
+  .containsFilter('elements.title', ['Warrior', 'Monk'])
```
Comment (Enngage) [inline doc/delivery.md]: @kentico-anthonym Technically, this method also contains 2 parameters :-) Its just that the second parameter expects an array to be passed.

File: doc/delivery.md [hunk 2]
```diff
@@ -186,7 +186,7 @@ deliveryClient.items<Movie>()
 
 This example returns all **Movie** content items whose **title** element is equal to **Warrior**. Filters are also considered query parameters and can be combined. See [Content filtering](https://developer.kenticocloud.com/v1/reference#content-filtering) in the Kentico Cloud API reference for more general examples.
 
-Supported filters:  `type`,  `types`, `allFilter`, `anyFilter`, `containsFilter`, `equalsFilter`, `greaterThanFilter`, `greaterThanOrEqualFilter`, `infilter`, `lessThanFilter`, `lessThanOrEqualFilter`, `rangeFilter`.
+Filters accepting two parameters:  `type`,  `types`, `equalsFilter`, `greaterThanFilter`, `greaterThanOrEqualFilter`, `lessThanFilter`, `lessThanOrEqualFilter`, `rangeFilter`.
```
Comment (Enngage) [inline doc/delivery.md]: @kentico-anthonym , I'm not sure I like this distinction. What I would possibly liked is to have something like https://github.com/Enngage/kentico-cloud-js/blob/master/doc/delivery.md#client-configuration - a table which could list all supported params as well as their possible input parameters. It's also possible there are (?) or will be more ways to call these methods and they could easily be incorporated in such a table if necessary.  Would you be able to create something like this?

Comment (kentico-anthonym) [reply doc/delivery.md]: Yeah I'm working on it.

**Comment by kentico-anthonym** [comment]
Omission of the parameters is what caused the confusion with the customer in the first place. I will update accordingly, but I feel like they should be at least visible since separating them into simple vs array type of filter is not a viable solution. Unless you feel like the KC docs filtering section is more clean. [Filtering](https://developer.kenticocloud.com/reference#content-filtering)

**Comment by Enngage** [comment]
Okay, thanks!

**Pull request merged**